src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/TupleTypeDecoder.vb

```diff
@@ -190,15 +186,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
                                      decodedArgs.ZipAsArray(type.TypeArgumentsCustomModifiers, Function(t, m) New TypeWithModifiers(t, m)),
                                      decodedArgs.SelectAsArray(Function(t) New TypeWithModifiers(t, Nothing)))
 
-                If (containerChanged) Then
+                If containerChanged Then
                     decodedType = decodedType.OriginalDefinition.AsMember(decodedContainingType)
                     ' If the type is nested, e.g. Outer(of T).Inner(of V), then Inner is definitely
                     ' not a tuple, since we know all tuple-compatible types (System.ValueTuple)
                     ' are not nested types. Thus, it is safe to return without checking if
                     ' Inner is a tuple.
-                    Return If(type.TypeParameters.IsEmpty,
-                                type,
-                                Construct(type, newTypeArgs))
+                    Return If(decodedType.TypeParameters.IsEmpty,
+                                decodedType,
+                                Construct(decodedType, newTypeArgs))
```

Comment:
There were failing tests

---
In reply to: [78046660](https://github.com/dotnet/roslyn/pull/13669#discussion_r78046660) [](ancestors = 78046660)